shared-helpers/src/locales/tl.json

```diff
@@ -340,10 +340,10 @@
   "authentication.signIn.changeYourPassword": "Maaari mong palitan ang iyong password",
   "authentication.signIn.enterLoginEmail": "Pakilagay ang iyong email sa pag-login",
   "authentication.signIn.enterLoginPassword": "Pakilagay ang iyong password sa pag-log in",
-  "authentication.signIn.enterValidEmailAndPasswordAndMFA": "Pakilagay ang tamang code.",
   "authentication.signIn.enterValidEmailAndPassword": "Pakilagay ang tamang email at password.",
-  "authentication.signIn.errorGenericMessage": "Pakisubukang muli_ o makipag-ugnayan sa suporta para humingi ng tulong.",
+  "authentication.signIn.enterValidEmailAndPasswordAndMFA": "Pakilagay ang tamang code.",
   "authentication.signIn.error": "Nagkaproblema sa iyong pag-sign in",
+  "authentication.signIn.errorGenericMessage": "Pakisubukang muli_ o makipag-ugnayan sa suporta para humingi ng tulong.",
```

Comment:
This was fixed recently in core, but somehow we've had a lot of underscores get in where commas are supposed to be, it was easy to find and place them with commas, there were only a handful of cases in tags where the underscore was needed.

Reply:
Ah yeah, I saw the convo about that and forgot to double-check here. Will take a look.